### PR TITLE
tm/ceph/resize support for non-persistent disks

### PR DESCRIPTION
**Pull request by atodorov-storpool**
The non-persistent disks has `CLONE=YES` attribute set.
This patch appends VM_ID and DISK_ID only when non-persistent disks are resized

### CODE REVIEW

**Comment by jfontan** [comment]
Tanks Anton and sorry for the delay. It was automatically closed because the branch was deleted.

@rsmontero this change should be added to master. If you're OK I can do it.

**Comment by jfontan** [comment]
Changed the base to master. It should be possible to merge in github.

File: src/tm_mad/ceph/resize [hunk 1]
```diff
@@ -60,14 +60,19 @@ while IFS= read -r -d '' element; do
...
 RBD_SRC="${XPATH_ELEMENTS[j++]}"
+CLONE="${XPATH_ELEMENTS[j++]}"
 CEPH_USER="${XPATH_ELEMENTS[j++]}"
 CEPH_CONF="${XPATH_ELEMENTS[j++]}"
 
-RBD_DST="${RBD_SRC}-${VM_ID}-${DISK_ID}"
+RBD_DST="$RBD_SRC"
+if [ "$CLONE" = "YES" ]; then
+    RBD_DST+="-${VM_ID}-${DISK_ID}"
+else
```
Comment (goberle) [inline src/tm_mad/ceph/resize]: This should be "fi" no ?

Comment (goberle) [inline src/tm_mad/ceph/resize]: Why "else" and not "fi" ? :)

Comment (atodorov-storpool) [inline src/tm_mad/ceph/resize]: yep. typo :/

**Comment by atodorov-storpool** [comment]
resolved in  commit c0e451c5e95cf91077fa379c355b3cbe3c058d26

**Pull request closed without merging**